src/applications/representative-appoint/tests/e2e/navigation/2122a.cypress.spec.js

```diff
@@ -0,0 +1,233 @@
+import * as h from '../../../utilities/e2eHelpers';
+import { ROUTES } from '../../../constants/routes';
+import attorney from '../../fixtures/data/attorney.json';
+
+describe('Unauthenticated', () => {
+  context('User is non-veteran', () => {
+    beforeEach(() => {
+      cy.intercept(
+        'GET',
+        '/representation_management/v0/original_entities?query=**',
+        attorney,
+      ).as('fetchRepresentatives');
+      cy.intercept('POST', '/representation_management/v0/pdf_generator2122a', {
+        statusCode: 200,
+      }).as('pdfGeneration');
+    });
+
+    it('navigates through the flow successfully', () => {
+      cy.visit(h.ROOT);
+
+      // INTRO
+      h.verifyUrl(ROUTES.INTRO);
+      cy.injectAxeThenAxeCheck();
+
+      cy.injectAxeThenAxeCheck();
```

Comment:
You can delete this axe check - `cy.injectAxeThenAxeCheck();` - no need to do two checks consecutively.